upgrade_tests/cql_tests.py

```diff
@@ -4122,6 +4122,7 @@ def test_static_columns_with_2i(self):
 
         cursor.execute("CREATE INDEX ON test(v)")
 
+        from_ver = float(self.UPGRADE_PATH.starting_meta.family)
```

Comment:
can't family contain stuff like `x`?  should we use LooseVersion like we do in other tests?

checking...

Reply:
just checked, the following are current family versions

```
family='2.1'
family='3.0'
family='3.11'
family='trunk'
```

so the last one would break as trunk isn't a valid float.  

What we can do is the following

after `self.prepare` do

```
initial_version = self.cluster.version()
```

then change the if check to

```
if initial_version  > LooseVersion('3.0'):
```

Reply:
I can try that out. I didn't know any upgrade tests start with trunk. Is that a thing?